tests/storage/test_delete.py

```diff
@@ -0,0 +1,79 @@
+import hashlib
+import os
+import glob
+import traceback
+import re
+import binascii
+import logging
+import sys
+sys.path.append("..")
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+import argparse
+
+
+extension = ".txt"
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Arguments', formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument( "--file_name", dest="file_name",
+                        required = True,
+                        help='Name of the file')
+    parser.add_argument("--dest", dest="path",
+                        required=True,
+                        help='Specify the path')
+    args = parser.parse_args()
+    return args
+
+
+class Delete():
+
+    def __init__(self, file_name, dest_path):
+        self.exit = 0
+        self.ref_timestamp = int(time.time())
+        self.storage_path = dest_path
+        self.ref_test_name = 'delete'
+        self.file_name = file_name
+        #self.parentDirectory = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+        self.root_file_path= os.path.join("/", dest_path)
+        self.file_path = os.path.join(self.root_file_path, file_name)
+        #self.file_path = os.path.join(self.parentDirectory, self.eos_path)
+        self.logger_folder = os.path.join(os.getcwd(), LOG_FOLDER)
+        self.log = Logger(os.path.join(self.logger_folder, self.ref_test_name +"_" + time.strftime("%Y-%m-%d_%H:%M:%S")+ LOG_EXTENSION))
```

Comment:
We agreed that the logging is going to be modular, right? Are you implementing that in a future PR?

Reply:
Yes